Parsing.php

```diff
@@ -0,0 +1,64 @@
+<?php
+declare(strict_types=1);
+
+
+/**
+ * Directory for file set in FILES_DIRECTORY
+ * Returns an array with the unchanged file lines
+ *
+ * @param string $filename
+ * @return array
+ */
+function fileToArray(string $filename): array
+{
+    $returnArr = [];
+    $filename  = FILES_DIRECTORY . $filename;
+    $f         = fopen($filename, 'r');
+    while ($line = fgets($f)) {
```

Comment:
if you use `fgetcsv` here, you can replace `fileToTableStructure` with this method.

Reply:
solved